src/jsontypes.rs

```diff
@@ -49,6 +50,8 @@ pub struct RawSourceMap {
     pub x_metro_module_paths: Option<Vec<String>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub x_facebook_sources: FacebookSources,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub debug_id: Option<DebugId>,
```

Comment:
I see the type is not `#[non_exhaustive]`, but I believe it is not public so this should be fine.